sip-core/src/main/java/de/ikor/sip/foundation/core/trace/SIPTraceTypeEnum.java

```diff
@@ -0,0 +1,17 @@
+package de.ikor.sip.foundation.core.trace;
+
+import java.util.function.BiConsumer;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public enum SIPTraceTypeEnum {
+  LOG("log", CustomTracer::logTrace),
+  MEMORY("memory", CustomTracer::storeInMemory);
+
+  public final String label;
+  private final BiConsumer<CustomTracer, String> consumer;
+
+  public void execute(CustomTracer customTracer, String out) {
```

Comment:
I like the pattern, but the Enum name feels wrong now.
When I see Type in the name, I expect it to do nothing.